cli_research_interactive.py

```
@@ -0,0 +1,317 @@
+            "summarization_model": "deepseek:deepseek-chat",
+            "summarization_model_max_tokens": 4096,
+            "research_model": model,
+            "research_model_max_tokens": 8192,
+            "compression_model": "deepseek:deepseek-chat",
+            "compression_model_max_tokens": 4096,
+            "final_report_model": model,
+            "final_report_model_max_tokens": 8192,
+        }
+    }
+
+    # 如果提供了文档路径，添加MCP配置
```

Ruff (0.13.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)